src/RwaConduit.sol

```diff
@@ -0,0 +1,100 @@
+pragma solidity 0.5.12;
+
+import "lib/dss-interfaces/src/dapp/DSTokenAbstract.sol";
+
+contract RwaConduit {
+    DSTokenAbstract public gov;
+    DSTokenAbstract public dai;
+    address public to;
+
+    event Push(address to, uint256 wad);
+
+    constructor(address _gov, address _dai, address _to) public {
+        gov = DSTokenAbstract(_gov);
+        dai = DSTokenAbstract(_dai);
+        to = _to;
+    }
+
+    function push() external {
+        require(gov.balanceOf(msg.sender) > 0);
+        emit Push(to, dai.balanceOf(address(this)));
```

Comment:
consider pulling dai.balanceOf into a memory variable to avoid two external calls here